recipes/ffmpeg/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: http://ffmpeg.zeranoe.com/blog/?p=300
+
+bash "%RECIPE_DIR%\mingw-w64-build-3.6.0.sh"
```

Comment:
Yeah, I thought it was a bit weird to. I think this was back when MinGW was used more. We could just download and install pre-built binaries for Windows.